web/src/features/menu/list/index.tsx

```diff
@@ -50,6 +50,7 @@ const ListMenu: React.FC = () => {
   };
 
   const moveMenu = (e: React.KeyboardEvent<HTMLDivElement>) => {
+    if (firstRenderRef.current) firstRenderRef.current = false
```

Comment:
Issue with this is that if the first button is a normal button the `onSelect` event won't fire.